### PR TITLE
Clean up & dependency update

### PR DESCRIPTION
**Pull request by kaelig**
- Update syntax to SassDoc 2.0
- Fix typos in readme
- Remove dependency to o-fonts
- ~~Introduce dependency to o-ft-typography for the colour use cases (@albertoelias not sure we should do that, what do you think?)~~

### CODE REVIEW

File: demos/links-card.html [hunk 1]
```diff
@@ -3,16 +3,17 @@
...
+	<meta name="viewport" content="initial-scale=1.0, width=device-width" />
+	<script src="//polyfill.webservices.ft.com/v1/polyfill.min.js"></script>
+	<style>body { margin: 0; } .demo-js .o--if-nojs { display: none; }</style>
+	<script>(function(d) { d.className = d.className + ' demo-js'; })(document.documentElement);</script>
+	<link rel="stylesheet" href="/bundles/css?modules=o-business-card:/demos/src/demo.scss,o-fonts@^1.5.0" />
 </head>
 <body class="">
-    <script>(function(d) {var b = d.getElementsByTagName('body')[0];b.className = b.className + ' demo-js';})(document);</script>
-    <div data-o-component="o-business-card" data-o-version="" class="o-business-card" itemscope itemtype="http://schema.org/Person">
+<div data-o-component="o-business-card" data-o-version="" class="o-business-card" itemscope itemtype="http://schema.org/Person">
```
Comment (AlbertoElias) [inline demos/links-card.html]: we can remove data-o-version

**Comment by AlbertoElias** [comment]
I don't think we should include o-ft-typography for a colour use case. And colour use cases are set in the module normally, I think there should at least be a default colour set.

**Comment by kaelig** [comment]
Thanks for the comment. I've removed the dependency and the module version.

File: src/scss/_variables.scss [hunk 2]
```diff
@@ -1,3 +1,10 @@
+////
+/// @group o-business-card
+/// @link http://registry.origami.ft.com/components/o-business-card
+////
+
 /// Silent mode
 /// @type Bool
 $o-business-card-is-silent: true !default;
+
+@include oColorsSetUseCase(o-business-card, border, pink-tint2);
```
Comment (AlbertoElias) [inline src/scss/_variables.scss]: You would need to set the background as well.

Comment (kaelig) [inline src/scss/_variables.scss]: There is a fallback in case the background isn't set, so not sure why we'd need to set it?

**Comment by AlbertoElias** [comment]
There are merge conflicts

**Comment by kaelig** [comment]
@AlbertoElias I fixed the merge conflicts, feel free to review again.

File: demos/bio-card.html [hunk 3]
```diff
@@ -8,17 +10,18 @@
 	<script src="//polyfill.webservices.ft.com/v1/polyfill.min.js?features=default"></script>
 	<style>body { margin: 0; } .demo-js .o--if-nojs { display: none; }</style>
 	<script>(function(d) { d.className = d.className + ' demo-js'; })(document.documentElement);</script>
-	<link rel="stylesheet" href="/bundles/css?modules=o-business-card:/demos/src/demo.scss,o-fonts@^1.4.0" />
+	<link rel="stylesheet" href="/bundles/css?modules=o-business-card:/demos/src/demo.scss,o-fonts@^1.5.0" />
 </head>
-<body>
-<div data-o-component="o-business-card" data-o-version="" class="o-business-card" itemscope itemtype="http://schema.org/Person">
-	<img src="http:&#x2F;&#x2F;image.webservices.ft.com&#x2F;v1&#x2F;images&#x2F;raw&#x2F;fthead:lucy-kellaway?source=docs&amp;format=png&amp;width=60" class="o-business-card__image--left" itemprop="image" />
+<body class="">
```
Comment (AlbertoElias) [inline demos/bio-card.html]: can you rebuild demos using latest obt beta?

Comment (kaelig) [inline demos/bio-card.html]: Done, thanks.

**Pull request merged**